db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/sqlgrammar.jj

```diff
@@ -2042,13 +2042,13 @@ public class SQLParser
      *      - the padding character
      * @exception StandardException  Thrown on error
      */
-    ValueNode getLeftOperatorNode(ValueNode stringValue, ValueNode length, ValueNode padding) throws StandardException
+    ValueNode getLeftOperatorNode(ValueNode stringValue, ValueNode length) throws StandardException
     {
         return (ValueNode) nodeFactory.getNode(
                 C_NodeTypes.LEFT_OPERATOR_NODE,
                 stringValue,
                 length,
-                padding,
+                null,
                 ReuseFactory.getInteger(TernaryOperatorNode.LEFT),
```

Comment:
LEFT now takes 2 arguments.  Should we make it a BinaryOperatorNode?

Reply:
Since LEFT() and RIGHT() are similar, please see my comment at: [https://github.com/splicemachine/spliceengine/pull/2746#discussion_r290761412](https://github.com/splicemachine/spliceengine/pull/2746#discussion_r290761412)

Reply:
OK